src/ophys_etl/schemas/_roi_schema.py

```diff
@@ -7,23 +7,55 @@ class ExtractROISchema(Schema):
     'rois' field
     """
 
-    id = Int(required=True,
-             description=("Unique ID of the ROI, get's overwritten writting "
-                          "to LIMS"))
-    x = Int(required=True,
+    id = Int(
+            required=True,
+            description=("Unique ID of the ROI, get's overwritten writting "
+                         "to LIMS"))
+    x = Int(
+            required=True,
             description="X location of top left corner of ROI in pixels")
-    y = Int(required=True,
+    y = Int(
+            required=True,
             description="Y location of top left corner of ROI in pixels")
-    width = Int(required=True,
-                description="Width of the ROI in pixels")
-    height = Int(required=True,
-                 description="Height of the ROI in pixels")
-    valid = Bool(required=True,
-                 description=("Boolean indicating if the ROI is a valid "
-                              "cell or not"))
-    mask = List(List(Bool), required=True,
-                description=("Bool nested list describing which pixels "
-                             "in the ROI area are part of the cell"))
+    width = Int(
+            required=True,
+            description="Width of the ROI in pixels")
+    height = Int(
+            required=True,
+            description="Height of the ROI in pixels")
+    mask = List(
+            List(Bool),
+            required=False,
+            description=("Bool nested list describing which pixels "
+                         "in the ROI area are part of the cell"))
+    mask_matrix = List(
+            List(Bool),
+            required=False,
+            description=("Bool nested list describing which pixels "
+                         "in the ROI area are part of the cell"))
+    valid = Bool(
+            required=False,
+            description=("Boolean indicating if the ROI is a valid "
+                         "cell or not"))
+    valid_roi = Bool(
```

Comment:
Same description as `valid`, is this intentional?

Reply:
explained below in the `post_load` but I will add some verbiage up here also.

Reply:
done